testapi.pm

```diff
@@ -47,6 +47,7 @@ our @EXPORT = qw($realname $username $password $serialdev %cmd %vars
   enter_cmd
   hold_key release_key
 
+  assert_screen check_screen assert_and_tclick save_screenshot
```

Comment:
you are adding a new test function but also duplicating other entries. Every function name should only be mentioned once in this list. Also, I don't think we need the function "assert_and_tclick" at all and test API extensions should not be taken lightly. I suggest to not extend the file testapi.pm at all also especially as your commit is called "Add test coverage for …", not "Add new test API function"

Reply:
Just seen your comments.Let me make those changes. Thanks